### PR TITLE
Typos from history of CPython

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -233,6 +233,7 @@ accurancy->accuracy
 accurarcy->accuracy
 accurary->accuracy, actuary,
 accuray->accuracy, actuary,
+accurs->occurs
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: This is a legit word per https://en.oxforddictionaries.com/definition/accurse  
What do think @larsoner ?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -2867,10 +2881,12 @@ colomns->columns
 colon-seperated->colon-separated
 colonizators->colonizers
 colorfull->colorful
+colorizoer->colorizer
 coloum->column
 coloumn->column
 coloumns->columns
 coloums->columns
+colours->colors
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: remove this because this is correct british spelling

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -6350,6 +6378,7 @@ expeption->exception
 experiance->experience
 experianced->experienced
 experiement->experiment
+experimentatin->experimentation
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: can also be `experimenting`

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -7261,6 +7295,7 @@ habeus->habeas
 Habsbourg->Habsburg
 hace->have
 hach->hatch, hack, hash,
+haders->headers
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Can also be `haters` :stuck_out_tongue_winking_eye:

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -9544,6 +9592,7 @@ meterologist->meteorologist
 meterology->meteorology
 methaphor->metaphor
 methaphors->metaphors
+methds->methods
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please also add `methd->method` above this, thx

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -9622,6 +9671,7 @@ mimimise->minimise, minimize,
 mimimize->minimize, minimise,
 mimimum->minimum
 mimimun->minimum
+miminal->minimal
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Please add `miminally->minimally` and `miminaly->minimally`

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -9790,6 +9841,7 @@ momemtn->moment
 momentarely->momentarily
 momento->memento
 momery->memory
+mone->mono
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: can be also `money`

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -15501,14 +15583,18 @@ truied->tried
...
 truned->turned
 trustworthyness->trustworthiness
 trustworty->trustworthy
 trustwortyness->trustworthiness
 tryed->tried
 tryes->tries
 tryig->trying
+tryinng->trying
 trys->tries
+tswitch->switch
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: not sure about this one

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -14799,6 +14802,7 @@ swicth->switch
 swicthed->switched
 swicthing->switching
 swiming->swimming
+switch->switch
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: please remove this incorrect entry

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (follower) [reply codespell_lib/data/dictionary.txt]: I may be misunderstanding your comment, but the link refers to `accurse` (with the last letter `e`) but the typo is for `accurs` (with last letter `s`).

`accurse` *is* a legit word but `accurs` is not.

Or did you mean that `accurse` could be a possible correction also?

While we are talking about code which might well be accursed, I think "occurs" would be more likely. :)

File: codespell_lib/data/dictionary.txt (hunk 9 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added

**Comment by luzpaz** [comment]
@Gelma LGTM, please rebase  
@peternewman green light from me

**Comment by Gelma** [comment]
> @Gelma LGTM, please rebase
> @peternewman green light from me

I'm going to do it right now

**Pull request closed without merging**